dockers/Screenshotter/screenshotter.js

```diff
@@ -202,6 +240,8 @@ function imageDimensions(img) {
 //////////////////////////////////////////////////////////////////////
 // Take the screenshots
 
+var countdown = listOfCases.length;
```

Comment:
Right.  I was kind of surprised that Selenium allows you schedule stuff in parallel.  It's nice that it queues the screenshot requests for us.